liblwgeom/lwgeom_transform.c

```diff
@@ -413,77 +246,11 @@ lwgeom_transform_from_str(LWGEOM *geom, const char* instr, const char* outstr)
 		lwerror("%s: Failed to transform", __func__);
 		return LW_FAILURE;
 	}
-
-	LWPROJ *lp = lwproj_from_PJ(pj, LW_FALSE);
-
 	int ret = lwgeom_transform(geom, lp);
-
-	proj_destroy(pj);
-	lwfree(lp);
-
+	proj_destroy(lp->pj);
```

Comment:
This is leaking `lp` as it's missing a `lwfree(lp);` right after this line. Not super important since the context will free it when running under PG, but an issue when not.